src/main/scala/mesosphere/marathon/api/akkahttp/PathMatchers.scala

```diff
@@ -0,0 +1,74 @@
+package mesosphere.marathon
+package api.akkahttp
+
+import akka.http.scaladsl.model.Uri.Path
+import mesosphere.marathon.core.task.Task
+import mesosphere.marathon.state.{ Group, PathId, RootGroup, Timestamp }
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.PathMatcher.{ Matched, Unmatched }
+import akka.http.scaladsl.server.PathMatcher1
+
+import scala.annotation.tailrec
+
+object PathMatchers {
+
+  /**
+    * Matches the remaining path and transforms it into task id
+    */
+  final val RemainingTaskId = Remaining.map(s => Task.Id(s))
```

Comment:
Is this a matcher as in `"apps" / appId / "tasks" / RemainingTaskId`?

Reply:
Yep, you can use it there.